README.adoc

```diff
@@ -238,10 +238,16 @@ fun `WHEN putting file on sftp THEN file is read and moved to a correct director
 ----
 
 === Absolute vs relative paths
-The SFTP Server Stub supports both absolute and relative paths (relative are interpreted as relative to user's
-home directory). However, in connection to flow testing you want to avoid using absolute paths, as the `readFiles`
-and `writeFiles` processors in Utilihive Flow Server interpret all paths as relative to user's home directory,
-whether they are prepended with `/` or not.
+The SFTP Server Stub supports both absolute and relative paths. Paths without `/`
+prefix are interpreted relative to `defaultDirectory` (`/`, unless overridden
+when initializing the context).
+
+In connection to flow testing you should keep in mind that the `readFiles` and
```

Comment:
I think the description here of the absolute/relative aspects of moveToFolder and errorFolder is incorrect?
If they are absolute they are interpreted as absolute in the part of the file system you have access too and relative to the "path" property when relative. 

I.e. given login dir "/" and path "/files/in". If moveToFolder is "/files/archive" that is the exact place in the file system where it will end up. Given "files/archive" it will end up relative to the "path":  "/files/in/files/archive".

Reply:
I have cut down on describing flow server behaviour, will leave it to the actual processor doc.